n4m-vimeo/utils.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+	isTokenValid: function(authToken) {
+		if (authToken.length <= 4) {
```

Comment:
Technically not a completely accurate way to check ... perhaps let's just check if it is null or not? I also added some regex parsing, so perhaps that's the logic we can adopt instead.

Reply:
I remember us settling on `<=4` as a reasonable check during an earlier iteration of this. 
Since the auth token is generated, not sure how we could use a regex to verify it's valid. Are there any redundant parts to Vimeo authentication credentials that we can filter for?

Reply:
The real way to check a token is to actually validate it against the Vimeo API, but it's not worth it. It's simpler to just check if it's not null and then when you try to load a video, Vimeo videos API endpoint will tell you if you provided an invalid token or not.